internal/backends/testing/ffi.rs

```diff
@@ -107,3 +107,16 @@ pub extern "C" fn slint_testing_element_bases(
         false
     }
 }
+
+#[no_mangle]
+pub extern "C" fn slint_testing_element_accessible_role(
```

Comment:
It's a bit overkill to expose a ffi function for that, when the code to call it from the vtable takes about as many lines. (that's what we do for the other accessible properties)

Reply:
Ok :)